write_toml() does not compile with MSVC

Environment:
```
Microsoft Visual Studio Community 2022 (64-bit) - Current
Version 17.14.29 (March 2026)
Compiler Version 19.44.35225 
```

Error:
```
glaze\toml\write.hpp(1104,64): error C2131: expression did not evaluate to a constant
```

Minimal example:
```
#include <glaze/toml.hpp>
#include <string>

struct Settings
{
    int value{42};
    std::string name{"test"};
};

template <>
struct glz::meta<Settings>
{
    using T = Settings;
    static constexpr auto value = object("value", &T::value, "name", &T::name);
};

int main()
{
    Settings s;
    std::string buffer;
    auto ec = glz::write_toml(s, buffer);
    return ec ? 1 : 0;
}
```

I've also attached the minimal CMake/vcpkg project.

[glaze_bug.zip](https://github.com/user-attachments/files/26619742/glaze_bug.zip)

--- a/README.md
+++ b/README.md
@@ -253,7 +253,7 @@ The buffer refills automatically during parsing, enabling reading of arbitrarily
 - Tested for both 64bit and 32bit
 - Supports both little-endian and big-endian systems
 
-[Actions](https://github.com/stephenberry/glaze/actions) build and test with [Clang](https://clang.llvm.org) (18+), [MSVC](https://visualstudio.microsoft.com/vs/features/cplusplus/) (2022), and [GCC](https://gcc.gnu.org) (13+) on apple, windows, and linux. Big-endian is tested via QEMU emulation on s390x.
+[Actions](https://github.com/stephenberry/glaze/actions) build and test with [Clang](https://clang.llvm.org) (18+), [MSVC](https://visualstudio.microsoft.com/vs/features/cplusplus/) (Visual Studio 2026 MSVC Build Tools 14.50), and [GCC](https://gcc.gnu.org) (13+) on apple, windows, and linux. Big-endian is tested via QEMU emulation on s390x.
 
 ![clang build](https://github.com/stephenberry/glaze/actions/workflows/clang.yml/badge.svg) ![gcc build](https://github.com/stephenberry/glaze/actions/workflows/gcc.yml/badge.svg) ![msvc build](https://github.com/stephenberry/glaze/actions/workflows/msvc.yml/badge.svg) 
 

